osbs/conf.py

```diff
@@ -194,6 +194,33 @@ def get_namespace(self):
         return self._get_value("namespace", self.conf_section, "namespace",
                                default=DEFAULT_NAMESPACE)
 
+    def get_flatpak(self):
+        return self._get_value("flatpak", self.conf_section, "flatpak",
+                               is_bool_val=True)
```

Comment:
This doesn't seem to get coverage during tests.

Reply:
Forcing it to be covered turned out that it wasn't getting called at all. I made it work because 'flatpak' and 'module' are analogous to 'git_url', 'git_branch' - it is additional specification of *what* to build. And those are parameters are in the config file, though not documented.